src/main/resources/application.yml

```diff
@@ -30,16 +30,26 @@ spring:
           starttls:
             enable: true
     confirmLinkIP: http://localhost:4200/
+  servlet:
+    multipart:
+      max-file-size: 50000KB
+      max-request-size: 50000KB
 jwt:
   bearer: Bearer
   # those are the expiration durations for jwt access and jwt refresh
   # introduced in milliseconds.
-  accessExpirationMs: 18000000
-  refreshExpirationMs: 30000000
+  accessExpirationMs: 80000000000
+  refreshExpirationMs: 90000000000
+
 
 springdoc:
   swagger-ui:
     path: /swagger-ui.html
 
+path:
+  basePackagePath: "avatars"
```

Comment:
Where is this package?

Reply:
this package is creating dinamicli with the first addPichture  request  on the app base package